docs/source/ops/connect/index.rst

```diff
@@ -3,8 +3,8 @@
 
 .. _ops-connect_index:
 
-Operating Daml
-==============
+Operating Daml Connect
+======================
```

Comment:
Why Daml Connect? We don't use this name anymore

Reply:
Mainly because it is in the `connect` sub directory.  As both pages that are contained within the `connect` sub folder (auth0 and helm) are both early access features we could do the following:
- Change the name of the directory to `early-access`
- Change the title in `index.rst` from `Operating Daml Connect` to `Early Access Features`
- In `operating-daml.rst` add the `ops/early-access/index` to the bottom (text displayed will be `Early Access Features`)

Reply:
This has been remove on purpose in a previous PR due to the renaming of the product. Please undo it.

Reply:
https://github.com/digital-asset/daml/pull/12668

Reply:
> Mainly because it is in the `connect` sub directory. As both pages that are contained within the `connect` sub folder (auth0 and helm) are both early access features we could do the following:
> 
> * Change the name of the directory to `early-access`
> * Change the title in `index.rst` from `Operating Daml Connect` to `Early Access Features`
> * In `operating-daml.rst` add the `ops/early-access/index` to the bottom (text displayed will be `Early Access Features`)

There's no need to make any change here, the change has been done purposefully to make sure links wouldn't break across versions. No need to change this.

Reply:
Reverted.